Projects/Features/PlaylistFeature/Sources/ViewControllers/PlayListDetailViewController.swift

```diff
@@ -50,399 +44,319 @@ public class PlayListDetailViewController: BaseViewController, ViewControllerFro
-        self.moreButton.isHidden = viewModel.type == .wmRecommend
-        self.editPlayListNameButton.isHidden = true
-    }
-
-    private func createDatasources() -> RxTableViewSectionedReloadDataSource<PlayListDetailSectionModel> {
-        let datasource = RxTableViewSectionedReloadDataSource<PlayListDetailSectionModel>(
-            configureCell: { [weak self] _, tableView, indexPath, model -> UITableViewCell in
-                guard let self = self else { return UITableViewCell() }
-
-                switch self.viewModel.type {
-                case .custom:
-                    guard let cell = tableView.dequeueReusableCell(
-                        withIdentifier: "PlayListTableViewCell",
-                        for: IndexPath(row: indexPath.row, section: 0)
-                    ) as? PlayListTableViewCell else {
-                        return UITableViewCell()
-                    }
-                    cell.update(model, self.input.state.value.isEditing, index: indexPath.row)
-                    cell.delegate = self
-                    return cell
+        playListImage.layer.cornerRadius = 12
 
-                case .wmRecommend:
-                    guard let cell = tableView.dequeueReusableCell(
-                        withIdentifier: "SongListCell",
-                        for: IndexPath(row: indexPath.row, section: 0)
-                    ) as? SongListCell else {
-                        return UITableViewCell()
-                    }
-                    cell.update(model)
-                    return cell
+        #warning("토큰 해결시 false 지우기")
+        moreButton.isHidden = false
+        // reactor?.type == .wmRecommend
+    }
 
-                case .none:
-                    return UITableViewCell()
-                }
+    override public func bind(reactor: PlaylistDetailReactor) {
+        super.bind(reactor: reactor)
 
-            },
-            canEditRowAtIndexPath: { _, _ -> Bool in
-                return true
-            },
-            canMoveRowAtIndexPath: { _, _ -> Bool in
-                return true
-            }
-        )
-        return datasource
-    }
+        tableView.rx.setDelegate(self)
+            .disposed(by: disposeBag)
 
-    private func bindRx() {
-        tableView.rx.setDelegate(self).disposed(by: disposeBag)
         tableView.register(
             UINib(nibName: "SongListCell", bundle: BaseFeatureResources.bundle),
             forCellReuseIdentifier: "SongListCell"
         )
+    }
 
-        // xib로 만든 UI를 컬렉션 뷰에서 사용하기 위해서는 등록이 필요
-        // 다른 모듈 시 번들 변경 Bundle.module 사용 X
+    override public func bindState(reactor: PlaylistDetailReactor) {
+        super.bindState(reactor: reactor)
 
-        output.dataSource
-            .skip(1)
-            .do(onNext: { [weak self] model in
-                guard let self = self else {
-                    return
-                }
-                self.activityIndicator.stopAnimating()
+        let currentState = reactor.state
```

Comment:
```suggestion
        let currentState = reactor.state.share(replay: 4)
```
이러면 옵져버블 스트림 하나로 여럿에 공유가능해요
ref : https://reactivex.io/documentation/operators/refcount.html

Reply:
그러니깐 4번을 한 흐름으로 처리한다는 뜻인가요 ?

Reply:
네네

Reply:
오 신기하다 ...